applications/crossbar/doc/port_requests.md

```diff
@@ -832,7 +920,7 @@ curl -v -X DELETE \
 ```shell
 curl -v -X PATCH \
     -H "X-Auth-Token: {AUTH_TOKEN}" \
-    http://{SERVER}:8000/v2/accounts/{ACCOUNT_ID}/port_requests/{PORT_REQUEST_ID}/submitted
+    http://{SERVER}:8000/v2/accounts/{ACCOUNT_ID}/port_requests/{PORT_REQUEST_ID}/submitted?reason=this+was+approved+by+Jane+Doe
```

Comment:
@fenollp why is this a query string param, should be in the payload (sorry if I missed this on the previous review)